docs/insights/project-insights.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Project Insights
+description: Project level insights are available as a new tab in every project so you can understand the performance of your projects across Channels, Environments, and Tenants.
```

Comment:
As per https://style.octopus.com/capitalization
I _think_ in this instance 'insights' is used as a common noun, not the proper name of the feature, so it can stay lower case.
```suggestion
description: Project level insights are available as a new tab in every project so you can understand the performance of your projects across channels, environments, and tenants.
```